### DISCUSSION TITLE
fail2ban-client status <JAIL> [FLAVOR]  <-- What is [FLAVOR]?

### DISCUSSION THREAD

**Discussion by frankmcc** [fail2ban/fail2ban#3233] [Q&A] [answered]
Excuse my ignorance, if there is documentation on this I have not found it.  It probably doesn't matter but I am so friggin curious I cannot stand it.  

What is the FLAVOR option?  Can it be helpful to me?  Can I just insert any Flavor?  Vanilla? Chocolate? Cookie-Dough? Ca I get sprinkles on top?

Is this feature simply undocumented or a mistake in the documentation?

My Monday is ruined now...

**Comment by sebres** [comment, marked as answer]
> Can I just insert any Flavor? 

Indeed you can (welcome with PR).

> Vanilla? Chocolate? Cookie-Dough?

An attempt to invoke `fail2ban-client status sshd vanilla` would just output basic status and write following warning to fail2ban.log:
```
WARNING Unsupported extended jail status flavor 'vanilla'. Supported: ['short', 'basic', 'cymru']
```

Initially it got introduced in #924 to output extended info for IP in status using Cymru. 
Later it was extended (in f381b982465e3e9bf58f565cfea006fedc1989f1) with option `short` to provide more clear output (#2819).

So at the moment it is `short`, `basic` and `cymru`.
And who knows which "flavors" we'd still support tomorrow.

> Is this feature simply undocumented or a mistake in the documentation?

Would you like to contribute and extend the documentation? Then welcome with PR.

**Reply by frankmcc** [reply]
Thank you for the explanation.  I can go on with Monday now.  :)

**Reply by melroy89** [reply]
PR would be welcome in the man page.

**Reply by pano9000** [reply]
I've submitted a PR: https://github.com/fail2ban/fail2ban/pull/3903
